### ISSUE TITLE
Plugin is not compatible with "woopl paypal payments"

### ISSUE THREAD

**Issue by procontentxyz** [jonas-hoebenreich/wc-donation-platform#15]
I have installed official paypal plugin by woocommerce:

https://wordpress.org/plugins/woocommerce-paypal-payments/

Donation plugin is not compatible.

........

Direct checkout is compatible @ checkout page itself, but in pop up its not compatible.

Please fix.

**Comment by jonas-hoebenreich** [comment]
It's working on my test site: https://testing.wcdp.jonh.eu/daily-donable-style-1-popup-1#wcdp-form
Could you please share more details what exactly is the problem?

**Comment by procontentxyz** [comment]
Which paypal plugin you are using

Are you using:

https://wordpress.org/plugins/woocommerce-paypal-payments/

Or some other?

**Comment by jonas-hoebenreich** [comment]
Yes, I am using https://wordpress.org/plugins/woocommerce-paypal-payments/ here
In fact, the site uses version 1.2.0-alpha of my plugin, but there have been no changes relevant to this case.

v1.2.0 has among other things an Express Donation (Apple Pay / Google Pay / PayPal) feature for the product page (e.g. here: https://testing.wcdp.jonh.eu/produkt/test-donable)

**Comment by jonas-hoebenreich** [comment]
Since you didn't give any details about your setup and the error, it's hard for me to figure out what exactly is going wrong. Do you have a test site where I can take a look at it?

**Comment by procontentxyz** [comment]
https://lockunlock.me/lockunlock-me-lock-sell-any-content-or-youtube-video-link/#wcdp-form  this is the error page... on test site


i have put below shortcode in short descrition of this product:

[wcdp_donation_form id="722" popup="1" button="0" style="5"]


paypal checkout is not happening

**Comment by jonas-hoebenreich** [comment]
OK, I know what the error is: on the product page the shortcode cannot be used. In your case, you can include the donation form only because you include it in the short description instead of the product description.
So in fact it is a bug. With the new version I will make sure that the shortcode cannot be used on the product page.

**Comment by procontentxyz** [comment]
so if i try to put it in description filed will it work?

actually i  want to implement pop up on product page....

but at the same time, i want paypal to work.

**Comment by jonas-hoebenreich** [comment]
There should be no problems with Style 4. However, with the other styles there is the conflict that a page cannot be a checkout and product page at the same time.

**Comment by procontentxyz** [comment]
style 5 is awesome, why its not supported for such scenerio ?

**Comment by jonas-hoebenreich** [comment]
Because one page cannot be product page & checkout page at the same time (What happens then you can see at your PayPal example)

**Comment by procontentxyz** [comment]
If so, then how its possible with style 4, as you said above

🤔

**Comment by jonas-hoebenreich** [comment]
It's not because it's trying to insert two forms (that case is already handled correctly), it's because PayPal and some other WooCommerce extensions can't handle that Product Page = Checkout Page.
Slution: Just use any other page / blog post to insert the form or settle for the provided design.
